File: codekata/absolute_beginner/Nth_term_of_the_series.py
# Nth term of the series
# get input from the user
n=int(input())
# define series function
def series():
# if condition check whether the given number is positive integer
    if(n>0):
# it generates series
     c=n*n
# return the c value
     return c
# elif condition check whether the given number is equal to zero
    elif(n==0):
# zero is stored in c
     c="0"
# return c value
     return c
# else condition check whether the given number is negative integer
    else:
# Error is stored in c
     c="Error"
# return c value
     return c
# call the series function and store the output in res
res=series()
# print the res
print(res)

File: codekata/javascript/absolute_beginner/Nth_term_of_the_series.js
// You are provided with a number "N", Find the Nth term of the series: 1, 4, 9, 25, 36, 49, 64, 81, .......
const readline = require('readline');
const inp = readline.createInterface({
  input: process.stdin
});
inp.on("line", (data)=>{
  if(data==0)
    console.log("0");
  else if(data>0)
    console.log(data*data);
  else
    console.log("Error");
  });
